package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@onchain-id/solidity",
-  "version": "0.4.1",
+  "version": "0.4.2",
```

Comment:
1.0.0 or 0.5.0, we changed interfaces. You should also export the IIentity.sol in the interfaces of `index.js`